src/TritonCTS/README.md

```diff
@@ -49,7 +49,7 @@ If this parameter is omitted, the code gets the value from ten times the height
 - ``clk_nets`` (optional) is a string containing the names of the clock roots. 
```

Comment:
All of the arguments keywords are missing the leading '-'.
The list showing optional versus mandatory is not a conventional way to describe arguments.
Optional ones are in brackets. See the resizer command description in the top level readme for an example.